src/core/FileManager.ts

```diff
@@ -80,7 +81,7 @@ export class FileManager extends ConfigProvider {
         return dependents.reduce((newContent, dependent) => {
             const thisDependent = this.assetsMap.get(dependent);
             if (thisDependent?.replacer) {
-                newContent = newContent.replace(dependent, thisDependent.replacer);
+                newContent = replaceAll(newContent, dependent, thisDependent.replacer);
```

Comment:
Actually, you do not need the `replaceAll` package or the function. You can simple change it back to what it was previously, with this slight modification.
```suggestion
                newContent = newContent.replace(new RegExp(dependent, 'g'), thisDependent.replacer);
```

And you can remove the dependency as well.

Reply:
You are completely correct of course. Elegant solution. I applied your suggestions.